### DISCUSSION TITLE
Upgrading V2 to V3

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#7986] [Help] [answered]
when following the steps provided in the documentations , it gives me this error which command `filament:install` is not found
![Screenshot from 2023-08-22 13-41-19](https://github.com/filamentphp/filament/assets/56083879/69c78d65-07e2-4b4d-ab2b-0c038a21819d)
what should i do ? to solve this issue

**Comment by leandrocfe** [comment]
did you run
`composer require filament/filament:"^3.0-stable" -W --no-update`
`composer update`
?

**Reply by Hegabovic** [reply]
yes when i ran this command it gave me these errors 

![Screenshot from 2023-08-22 13-55-38](https://github.com/filamentphp/filament/assets/56083879/8221b51a-46ee-40d9-9bd6-5729320a7404)

i need the composer.lock to stay when the current version for all other packages which are not related to filament

**Reply by leandrocfe** [reply, marked as answer]
make sure the plugins installed in your project support filament v3.
for example, [filament-browser](https://github.com/3x1io/filament-browser/blob/main/composer.json) doesn't support v3
*Some plugins you're using may not be available in v3 just yet. You could temporarily remove them from your composer.json file until they've been upgraded, replace them with a similar plugins that are v3-compatible, wait for the plugins to be upgraded before upgrading your app, or even write PRs to help the authors upgrade them.*

**Reply by Hegabovic** [reply]
thanks a lot for fast responding
